developer-docs-site/docs/tutorials/index.md

```diff
@@ -5,28 +5,43 @@ slug: "aptos-quickstarts"
 
 # Developer Tutorials
 
-If you are new to the Aptos blockchain, begin with these quickstarts before you get into in-depth development. These tutorials will help you become familiar with how to develop for the Aptos blockchain using the Aptos SDK. 
+If you are new to the Aptos blockchain, begin with these quickstarts before you get into in-depth development. These tutorials will help you become familiar with how to develop for the Aptos blockchain using the Aptos SDK.
+
+### Install macOS prerequisites
+
+If running macOS, install the following packages in the order specified to take these tutorials:
+
+1. **Homebrew**: [https://brew.sh/](https://brew.sh/)
+1. **Node.js**: Install [Node.js](https://nodejs.org/en/download/), which will install `npm` and `npx`, by executing the below command on your Terminal:
+    ```bash
+    brew install node
+    ```
+1. **Yarn**: Install the latest [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) by executing the below command on your Terminal:
+    ```bash
+    brew install yarn
+    ```
+1. **Poetry**: Install Poetry from [https://python-poetry.org/docs/#installation](https://python-poetry.org/docs/#installation).
```

Comment:
Theres are only for the SDK, I guess the tutorials all below are for the typescript SDK?

We probably should improve this later based on which language we do, whether we split it here or later.